optimize/util/optimize-commons/src/main/java/io/camunda/optimize/service/util/configuration/security/ResponseHeadersConfiguration.java

```diff
@@ -139,7 +145,11 @@ public Map<String, String> getHeadersWithValues() {
       headers.put(HttpHeaders.X_XSS_PROTECTION, getXsssProtection());
     }
 
-    if (getXContentTypeOptions()) {
+    Boolean xContentTypeOptions = getXContentTypeOptions();
+    if (xContentTypeOptions == null) {
+      xContentTypeOptions = false;
+    }
+    if (xContentTypeOptions) {
```

Comment:
Same here. Modify `getXContentTypeOptions()` to return a `boolean` so we can avoid `null` checks altogether. The getter is also used in two places, here and `toString()`